src/cmd/create.go

```diff
@@ -95,6 +103,10 @@ func init() {
 }
 
 func create(cmd *cobra.Command, args []string) error {
+	// This regex filters out strings which are not valid hostnames, according to RFC-1123.
+	// Source: https://stackoverflow.com/a/106223
+	var hostnameRegexp = regexp.MustCompile("^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])$")
```

Comment:
Do we really need to have this regular expression directly in the code?  Can't we use some Go package for it?